examples/tree_1d_dgsem/elixir_shallowwater_parabolic_bowl.jl

```diff
@@ -0,0 +1,109 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations1D(gravity_constant=9.81)
+cfl = 0.3
+
+
+function parabolic_bowl_analytic_1D_H(gravity, x,t)
+  a = 1
+  h_0 = 0.1
+  σ = 0.5
+  ω = sqrt(2*gravity*h_0)/a
+
+  H = σ * h_0/a^2 * (2*x[1]*cos(ω*t) - σ) + h_0
+  return H
+end
+
+# Implemented based on Wintermeyer (here seen in 2D)
+function initial_condition_parabolic_bowl(x, t, equations:: ShallowWaterEquations1D)
+  a = 1
+  h_0 = 0.1
+  σ = 0.5
+  ω = sqrt(2*equations.gravity*h_0)/a
+
+  v = -σ*ω*sin(ω*t)
+
+  b = h_0 * x[1]^2/a^2
+
+  H = max(b, parabolic_bowl_analytic_1D_H(equations.gravity, x, t))
```

Comment:
```suggestion
  sigma = 0.5
  ω = sqrt(2 * equations.gravity * h_0) / a

  v = -sigma * ω * sin(ω * t)

  b = h_0 * x[1]^2 / a^2

  H = max(b, parabolic_bowl_analytic_1D_H(equations.gravity, x, t))
```
We use `ω` other places for frequency so it can get "grandfathered" in. But I would change to sigma.